src/index.js

```diff
@@ -985,18 +1017,20 @@ export default function (babel) {
         }
       },
 
-      Function(path) {
-        if (path.node.kind === "constructor" || path.node.kind === "set") return;
+      Function: {
+        exit(path) {
```

Comment:
I am not actually 100% sure it is necessary to make this an `exit` visitor. Can anyone think of a test case that would exercise this?